8/community/run.sh

```diff
@@ -1,8 +1,8 @@
 #!/usr/bin/env bash
 
-set -e
+set -eou pipefail
```

Comment:
I'm a bit surprised this worked, since `pipefail` is a parameter of `-o`. I would write `set -euo pipefail` because it's less surprising.